### PR TITLE
OSDOCS-12743: added product compliance link to docs

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.12+ (except 4.13)

Issue:
https://issues.redhat.com/browse/OSDOCS-12743

Link to docs preview:
[85308--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-release-notes.html](https://85308--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-release-notes.html)
[85308--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/compliance_operator/co-overview.html](https://85308--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/compliance_operator/co-overview.html)
[85308--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/compliance_operator/co-scans/compliance-operator-supported-profiles.html](https://85308--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/compliance_operator/co-scans/compliance-operator-supported-profiles.html)
[85308--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/compliance_operator/co-support.html](https://85308--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/compliance_operator/co-support.html)

Additional information:
QE review not needed as this does not change the meaning of the docs.

### CODE REVIEW

**Review by rhmdnd — APPROVED** [review]
/lgtm

File: security/compliance_operator/co-overview.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ remediations, but actual compliance is your responsibility. You are required to
 work with an authorized auditor to achieve compliance with a standard. For the
 latest updates, see the
 xref:../../security/compliance_operator/compliance-operator-release-notes.adoc#compliance-operator-release-notes[Compliance
-Operator release notes]
+Operator release notes]. For more information on compliance support for all Red Hat products, see link:https://access.redhat.com/compliance[Product Compliance].
```
Comment (mburke5678) [inline security/compliance_operator/co-overview.adoc:21]: Per the [RH Supplementary Style Guide](https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces):

```suggestion
Operator release notes]. For more information on compliance support for all Red{nbsp}Hat products, see link:https://access.redhat.com/compliance[Product Compliance].
```

File: security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc [hunk 2]
```diff
@@ -17,6 +17,8 @@ Joint Authorization Board (JAB), or other industry recognized regulatory
 authority to assess your environment. You are required to work with an
 authorized auditor to achieve compliance with a standard.
 
+For more information on compliance support for all Red Hat products, see link:https://access.redhat.com/compliance[Product Compliance].
```
Comment (mburke5678) [inline security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc:20]:
```suggestion
For more information on compliance support for all Red{nbsp}Hat products, see link:https://access.redhat.com/compliance[Product Compliance].
```

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 3]
```diff
@@ -15,6 +15,8 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/co-management/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+For more information on compliance support for all Red Hat products, see link:https://access.redhat.com/compliance[Product Compliance].
```
Comment (mburke5678) [inline security/compliance_operator/compliance-operator-release-notes.adoc:18]:
```suggestion
For more information on compliance support for all Red{nbsp}Hat products, see link:https://access.redhat.com/compliance[Product Compliance].
```

**Comment by sheriff-rh** [comment]
Merging.

**Pull request merged**